boa/src/context.rs

```diff
@@ -543,10 +561,28 @@ impl Context {
         length: usize,
         body: NativeFunction,
     ) -> Result<()> {
-        let function = FunctionBuilder::new(self, body)
+        let function = FunctionBuilder::native(self, body)
+            .name(name)
+            .length(length)
+            .constructable(true)
+            .build();
+
+        self.global_object().insert_property(
+            name,
+            function,
+            Attribute::WRITABLE | Attribute::NON_ENUMERABLE | Attribute::CONFIGURABLE,
+        );
+        Ok(())
+    }
+
+    #[inline]
+    pub fn register_global_closure<F>(&mut self, name: &str, length: usize, body: F) -> Result<()>
```

Comment:
some docs above this like register_global_function has would be useful. Just to say why it exists and what usecase it may solve

Reply:
I will add documentation, I just wanted feedback first on the API